.github/pull-request-template.md

```diff
@@ -0,0 +1,21 @@
+### Mandatory Checklist
+*please replace the blank with 'x' mark in [ ], if you've completed it*
+- [ ] Passed all unit tests
+- [ ] Added test cases for the new features
+- [ ] The code coverage is higher than 80% 
+- [ ] Checked if README should be updated
+
+
+## 1. General Description
+* {your contribution}
+* {links to design docs, confluence pages, etc}
+*
+
+## 2. Related JIRA Tasks & GitHub Issues
+* https://code.sec.samsung.net/jira/browse/{JIRA}
```

Comment:
Since this is not related to Samsung I suggest to remove it.
```suggestion
```

Reply:
> Since this is not related to Samsung I suggest to remove it.

Nice Catch. Updated.